lib/stat_tracker.rb

```diff
@@ -16,12 +19,14 @@ def self.from_csv(locations_params)
 
     games_array = []
     CSV.foreach(games_file, headers: true, header_converters: :symbol) do |row|
-      games_array << row
+      game = Game.new(row)
+      games_array << game
```

Comment:
now with the game and team class we can create arrays of game and team instances with better searching capabilities